components/eamxx/cime_config/eamxx_buildnml.py

```diff
@@ -249,6 +249,63 @@ def _dict_representer(dumper, data):
     else:
         return yaml.dump(data, item, OrderedDumper, **kwds)
 
+###############################################################################
+def evaluate_selector(sel_name, sel_value, ez_selectors, case, child_name):
+###############################################################################
+    # Parse and and ors into separate statements
+    and_syntax = " @@ "
```

Comment:
How come not ` && `?

Reply:
I explained this in the documentation I added to the XML file but that came at the end of the PR, so easy to miss. The issue is that `&` is a reserved character in XML, so the only way to have it be ` && ` is ` &amp;&amp; `, not very pretty.

Reply:
What if we resorted to ` AND ` and ` OR `?

Reply:
We could, but I think we'd be more likely to have an unintended hit on those.